src/auro-input.js

```diff
@@ -28,7 +28,7 @@ export default class AuroInput extends BaseInput {
     /**
      * @private Boolean value to determine which password icon to show
      */
-    this.showPassword = false;
+    this._showPassword = false;
```

Comment:
So.... the `_`

This is not really a convention we are using anywhere else. 

![](https://media.giphy.com/media/IepriDYu8gcidIVZPB/giphy.gif)

Reply:
I don't think we need to make that standard. Old legacy C# conventions don't need to follow us here IMO.

Reply:
I followed the example in the [Lit docs](https://lit.dev/docs/components/properties/#public-properties-and-internal-state) on internal reactive state a little too closely. I'll remove the underscore.